### ISSUE TITLE
POST /projections/continuous returns 500 instead of 409

### ISSUE THREAD

**Issue by dnagir** [EventStore/EventStore#1905]
## Use case

We run the application using `docker-compose` which brings up EventStore along with multiple other containers.

The other containers are running "projectors" that lazily create an EventStore projection `ignore-system-events` (just filter out `$...` events and put them into a designated stream).

## Problem

Normally, when EventStore is running and we attempt to create a new projection (via HTTP) it will return either:

- 201 on success
- 409 if it already exists

But when the ES is still starting, it returns a 500 instead of 409:

```
#<HTTP::Response/1.1 500 Internal Server Error ...>: An item with the same key has already been added. Key: ignore-system-events
```

It is expected that it would consistently return a 409 and not a 500 _sometimes_.

The path for reference:
`projections/continuous?name=ignore-system-events&emit=yes&checkpoints=yes&enabled=yes&trackemittedstreams=no`

## Possible workarounds
- sleep between concurrent projection upserts
- extract the projection creation out of the application and manually create
- on 500

## Full Log

```
eventstore_1          | [00001,01,07:59:28.667] Quorum size set to 1
eventstore_1          | [00001,01,07:59:28.697] Cannot find plugins path: "/usr/share/eventstore/plugins"
eventstore_1          | [00001,01,07:59:28.844] "INSTANCE ID:"            61217f17-bcbc-4e4d-a56e-50b84d1fde11
eventstore_1          | [00001,01,07:59:28.844] "DATABASE:"               "/var/lib/eventstore"
eventstore_1          | [00001,01,07:59:28.844] "WRITER CHECKPOINT:"      0 (0x0)
eventstore_1          | [00001,01,07:59:28.861] "CHASER CHECKPOINT:"      0 (0x0)
eventstore_1          | [00001,01,07:59:28.861] "EPOCH CHECKPOINT:"       -1 (0xFFFFFFFFFFFFFFFF)
eventstore_1          | [00001,01,07:59:28.861] "TRUNCATE CHECKPOINT:"    -1 (0xFFFFFFFFFFFFFFFF)
eventstore_1          | [00001,01,07:59:29.191] MessageHierarchy initialization took 00:00:00.3009903.
eventstore_1          | [00001,01,07:59:29.424] CACHED TFChunk #0-0 (chunk-000000.000000) in 00:00:00.0075240.
eventstore_1          | [00001,01,07:59:29.668] Starting MiniWeb for "/web/es/js/projections" ==> "/usr/share/eventstore/projections"
eventstore_1          | [00001,01,07:59:29.668] Starting MiniWeb for "/web/es/js/projections/v8/Prelude" ==> "/usr/share/eventstore/Prelude"
eventstore_1          | [00001,01,07:59:29.668] Binding MiniWeb to "/web/es/js/projections/{*remaining_path}"
eventstore_1          | [00001,01,07:59:29.668] Binding MiniWeb to "/web/es/js/projections/v8/Prelude/{*remaining_path}"
eventstore_1          | [00001,01,07:59:29.693] Starting MiniWeb for "/web" ==> "/usr/share/eventstore/clusternode-web"
eventstore_1          | [00001,01,07:59:29.693] Binding MiniWeb to "/web/{*remaining_path}"
eventstore_1          | [00001,14,07:59:29.702] ========== [0.0.0.0:2112] SYSTEM INIT...
eventstore_1          | [00001,14,07:59:29.702] Starting "Normal" TCP listening on TCP endpoint: 0.0.0.0:1113.
eventstore_1          | [00001,15,07:59:29.714] TableIndex initialization...
eventstore_1          | [00001,14,07:59:29.727] Starting HTTP server on ["http://*:2113/"]...
eventstore_1          | [00001,15,07:59:29.734] ReadIndex building...
eventstore_1          | [00001,14,07:59:29.734] HTTP server is up and listening on ["http://*:2113/"]
eventstore_1          | [00001,15,07:59:29.734] ReadIndex rebuilding done: total processed 0 records, time elapsed: 00:00:00.0033120.
eventstore_1          | [00001,14,07:59:29.734] ========== [0.0.0.0:2112] Service '"StorageReader"' initialized.
eventstore_1          | [00001,14,07:59:29.734] ========== [0.0.0.0:2112] Service '"StorageWriter"' initialized.
eventstore_1          | [00001,14,07:59:29.765] ========== [0.0.0.0:2112] Service '"StorageChaser"' initialized.
eventstore_1          | [00001,14,07:59:29.765] ========== [0.0.0.0:2112] SYSTEM START...
eventstore_1          | [00001,14,07:59:29.779] ========== [0.0.0.0:2112] IS UNKNOWN...
eventstore_1          | [00001,05,07:59:29.782] Persistent subscriptions received state change to Unknown. Stopping listening
eventstore_1          | [00001,14,07:59:29.782] ELECTIONS: STARTING ELECTIONS.
eventstore_1          | [00001,14,07:59:29.782] ELECTIONS: (V=0) SHIFT TO LEADER ELECTION.
eventstore_1          | [00001,14,07:59:29.782] ELECTIONS: (V=0) VIEWCHANGE FROM [0.0.0.0:2112, {61217f17-bcbc-4e4d-a56e-50b84d1fde11}].
eventstore_1          | [00001,14,07:59:29.782] ELECTIONS: (V=0) MAJORITY OF VIEWCHANGE.
eventstore_1          | [00001,14,07:59:29.793] ELECTIONS: (V=0) SHIFT TO PREPARE PHASE.
eventstore_1          | [00001,14,07:59:29.793] ELECTIONS: (V=0) PREPARE_OK FROM "[0.0.0.0:2112,{61217f17-bcbc-4e4d-a56e-50b84d1fde11}](L=-1,W=0,C=0,E-1@-1:{00000000-0000-0000-0000-000000000000})".
eventstore_1          | [00001,14,07:59:29.793] ELECTIONS: (V=0) SHIFT TO REG_LEADER.
eventstore_1          | [00001,14,07:59:29.823] ELECTIONS: (V=0) SENDING PROPOSAL CANDIDATE: "[0.0.0.0:2112,{61217f17-bcbc-4e4d-a56e-50b84d1fde11}](L=-1,W=0,C=0,E-1@-1:{00000000-0000-0000-0000-000000000000})", ME: "[0.0.0.0:2112,{61217f17-bcbc-4e4d-a56e-50b84d1fde11}](L=-1,W=0,C=0,E-1@-1:{00000000-0000-0000-0000-000000000000})".
eventstore_1          | [00001,14,07:59:29.823] ELECTIONS: (V=0) ACCEPT FROM [0.0.0.0:2112,{61217f17-bcbc-4e4d-a56e-50b84d1fde11}] M=[0.0.0.0:2112,{61217f17-bcbc-4e4d-a56e-50b84d1fde11}]).
eventstore_1          | [00001,14,07:59:29.823] ELECTIONS: (V=0) DONE. ELECTED MASTER = "[0.0.0.0:2112,{61217f17-bcbc-4e4d-a56e-50b84d1fde11}](L=-1,W=0,C=0,E-1@-1:{00000000-0000-0000-0000-000000000000})". ME="[0.0.0.0:2112,{61217f17-bcbc-4e4d-a56e-50b84d1fde11}](L=-1,W=0,C=0,E-1@-1:{00000000-0000-0000-0000-000000000000})".
eventstore_1          | [00001,14,07:59:29.834] ========== [0.0.0.0:2112] PRE-MASTER STATE, WAITING FOR CHASER TO CATCH UP...
eventstore_1          | [00001,09,07:59:29.834] Persistent subscriptions received state change to PreMaster. Stopping listening
eventstore_1          | [00001,14,07:59:29.842] ========== [0.0.0.0:2112] IS MASTER... SPARTA!
eventstore_1          | [00001,09,07:59:29.842] Persistent subscriptions Became Master so now handling subscriptions
eventstore_1          | [00001,14,07:59:29.842] Searching for incomplete scavenges on node "0.0.0.0:2113".
eventstore_1          | [00001,14,07:59:29.942] SLOW BUS MSG ["MainBus"]: "ReadStreamEventsBackwardCompleted" - 84ms. Handler: "RequestResponseDispatcher`2".
eventstore_1          | [00001,14,07:59:29.942] SLOW QUEUE MSG ["MainQueue"]: "ReadStreamEventsBackwardCompleted" - 85ms. Q: 4/6.
eventstore_1          | [00001,14,07:59:29.942] Setting max age for the "$scavenges" stream to 30.00:00:00.
eventstore_1          | [00001,14,07:59:29.955] No incomplete scavenges found on node "0.0.0.0:2113".
eventstore_1          | [00001,12,07:59:30.103] === Writing E0@0:{c571c6ae-bcc5-4099-a411-fdeb1b5f009f} (previous epoch at -1).
eventstore_1          | [00001,12,07:59:30.103] === Update Last Epoch E0@0:{c571c6ae-bcc5-4099-a411-fdeb1b5f009f} (previous epoch at -1).
eventstore_1          | [00001,19,07:59:30.277] Created stats stream '"$stats-0.0.0.0:2113"', code = Success
eventstore_1          | [00001,14,07:59:30.332] SLOW QUEUE MSG ["MainQueue"]: "WriteEventsCompleted" - 50ms. Q: 4/5.
eventstore_1          | [00001,14,07:59:30.384] 'ops' user account has been created.
eventstore_1          | [00001,14,07:59:30.384] 'admin' user account has been created.
eventstore_1          | [00001,14,07:59:30.393] 'ops' user added to $users.
eventstore_1          | [00001,14,07:59:30.393] 'admin' user added to $users.
eventstore_1          | [00001,21,07:59:30.404] PROJECTIONS: Starting Projections Core Coordinator. (Node State : Master)
eventstore_1          | [00001,21,07:59:30.404] PROJECTIONS: Starting Projections Manager. (Node State : Master)
eventstore_1          | [00001,21,07:59:30.404] PROJECTIONS: Starting Projection Manager Response Reader (reads from $projections-$master)
eventstore_1          | [00001,22,07:59:30.419] PROJECTIONS: Starting Projection Core Reader (reads from $projections-$"cb6f4bdd33d3486597d83def8bd6ae6d")
eventstore_1          | [00001,24,07:59:30.419] PROJECTIONS: Starting Projection Core Reader (reads from $projections-$"38062e7e582a4c3b83dd71055dd8d4af")
eventstore_1          | [00001,23,07:59:30.419] PROJECTIONS: Starting Projection Core Reader (reads from $projections-$"5a38858005b4472cae895ac110875211")
eventstore_1          | [00001,21,07:59:30.425] PROJECTIONS: Resetting Worker Writer
eventstore_1          | [00001,22,07:59:30.425] PROJECTIONS: Resetting Master Writer
eventstore_1          | [00001,23,07:59:30.425] PROJECTIONS: Resetting Master Writer
eventstore_1          | [00001,21,07:59:30.425] PROJECTIONS: SubComponent Started: "EventReaderCoreService"
eventstore_1          | [00001,21,07:59:30.425] PROJECTIONS: SubComponent Started: "EventReaderCoreService"
eventstore_1          | [00001,21,07:59:30.425] PROJECTIONS: SubComponent Started: "EventReaderCoreService"
eventstore_1          | [00001,21,07:59:30.425] PROJECTIONS: SubComponent Started: "ProjectionCoreService"
eventstore_1          | [00001,24,07:59:30.425] PROJECTIONS: Resetting Master Writer
eventstore_1          | [00001,21,07:59:30.425] PROJECTIONS: SubComponent Started: "ProjectionCoreService"
eventstore_1          | [00001,21,07:59:30.425] PROJECTIONS: SubComponent Started: "ProjectionCoreService"
eventstore_1          | [00001,21,07:59:30.453] PROJECTIONS: Finished Starting Projection Manager Response Reader (reads from $projections-$master)
eventstore_1          | [00001,22,07:59:30.453] PROJECTIONS: Finished Starting Projection Core Reader (reads from $projections-$"cb6f4bdd33d3486597d83def8bd6ae6d")
eventstore_1          | [00001,23,07:59:30.462] PROJECTIONS: Finished Starting Projection Core Reader (reads from $projections-$"5a38858005b4472cae895ac110875211")
eventstore_1          | [00001,24,07:59:30.453] PROJECTIONS: Finished Starting Projection Core Reader (reads from $projections-$"38062e7e582a4c3b83dd71055dd8d4af")
eventstore_1          | [00001,21,07:59:30.462] PROJECTIONS: SubComponent Started: "ProjectionCoreServiceCommandReader"
eventstore_1          | [00001,21,07:59:30.462] PROJECTIONS: SubComponent Started: "ProjectionCoreServiceCommandReader"
eventstore_1          | [00001,21,07:59:30.462] PROJECTIONS: SubComponent Started: "ProjectionCoreServiceCommandReader"
eventstore_1          | [00001,24,07:59:30.462] PROJECTIONS: Starting read "$projections-$control-c571c6ae-bcc5-4099-a411-fdeb1b5f009f"
eventstore_1          | [00001,22,07:59:30.462] PROJECTIONS: Starting read "$projections-$control-c571c6ae-bcc5-4099-a411-fdeb1b5f009f"
eventstore_1          | [00001,23,07:59:30.462] PROJECTIONS: Starting read "$projections-$control-c571c6ae-bcc5-4099-a411-fdeb1b5f009f"
eventstore_1          | [00001,21,07:59:30.478] PROJECTIONS: Response received: 0@"$response-reader-starting"
eventstore_1          | [00001,21,07:59:30.493] PROJECTIONS: Response received: 1@"$projection-worker-started"
eventstore_1          | [00001,21,07:59:30.493] PROJECTIONS: Response received: 2@"$projection-worker-started"
eventstore_1          | [00001,21,07:59:30.493] PROJECTIONS: Response received: 3@"$projection-worker-started"
eventstore_1          | [00001,21,07:59:30.503] PROJECTIONS: Reading Existing Projections from "$projections-$all"
eventstore_1          | [00001,21,07:59:30.503] PROJECTIONS: No projections were found in "$projections-$all", starting from empty stream
eventstore_1          | [00001,14,07:59:30.519] ========== [0.0.0.0:2112] Sub System '"Projections"' initialized.
eventstore_1          | [00001,21,07:59:30.543] Adding projection 137f8fc1-d696-4f75-a2f4-586a43b1b1b2@"$streams" to list
eventstore_1          | [00001,21,07:59:30.558] PROJECTIONS: Scheduling the writing of "$create-and-prepare" to "$projections-$cb6f4bdd-33d3-4865-97d8-3def8bd6ae6d". Current status of Writer: Busy: false
eventstore_1          | [00001,21,07:59:30.581] Adding projection 1d7b7978-0cff-4cb4-84ee-38d54f33582a@"$stream_by_category" to list
eventstore_1          | [00001,21,07:59:30.581] Adding projection 2ae5d9fd-e8c8-4117-bbe5-bb49124b375e@"$by_category" to list
eventstore_1          | [00001,21,07:59:30.583] Adding projection c8abdc37-651b-4fc8-9f68-3d9f1a5fba08@"$by_event_type" to list
eventstore_1          | [00001,21,07:59:30.583] Adding projection 1e66aa1f-88cf-4ea9-a152-bdd8588d9da9@"$by_correlation_id" to list
eventstore_1          | [00001,21,07:59:30.583] PROJECTIONS: Scheduling the writing of "$create-and-prepare" to "$projections-$5a388580-05b4-472c-ae89-5ac110875211". Current status of Writer: Busy: false
eventstore_1          | [00001,21,07:59:30.583] PROJECTIONS: Scheduling the writing of "$create-and-prepare" to "$projections-$38062e7e-582a-4c3b-83dd-71055dd8d4af". Current status of Writer: Busy: false
eventstore_1          | [00001,21,07:59:30.583] PROJECTIONS: Scheduling the writing of "$create-and-prepare" to "$projections-$cb6f4bdd-33d3-4865-97d8-3def8bd6ae6d". Current status of Writer: Busy: true
eventstore_1          | [00001,21,07:59:30.583] PROJECTIONS: Scheduling the writing of "$create-and-prepare" to "$projections-$5a388580-05b4-472c-ae89-5ac110875211". Current status of Writer: Busy: true
eventstore_1          | [00001,21,07:59:30.583] PROJECTIONS: Finished writing events to "$projections-$cb6f4bdd33d3486597d83def8bd6ae6d": "$create-and-prepare"
eventstore_1          | [00001,21,07:59:30.583] PROJECTIONS: Finished writing events to "$projections-$5a38858005b4472cae895ac110875211": "$create-and-prepare"
eventstore_1          | [00001,21,07:59:30.583] PROJECTIONS: Finished writing events to "$projections-$38062e7e582a4c3b83dd71055dd8d4af": "$create-and-prepare"
eventstore_1          | [00001,21,07:59:30.592] PROJECTIONS: Finished writing events to "$projections-$cb6f4bdd33d3486597d83def8bd6ae6d": "$create-and-prepare"
eventstore_1          | [00001,22,07:59:30.592] PROJECTIONS: Command received: 0@"$create-and-prepare"
eventstore_1          | [00001,23,07:59:30.592] PROJECTIONS: Command received: 0@"$create-and-prepare"
eventstore_1          | [00001,24,07:59:30.592] PROJECTIONS: Command received: 0@"$create-and-prepare"
eventstore_1          | [00001,21,07:59:30.592] PROJECTIONS: Finished writing events to "$projections-$5a38858005b4472cae895ac110875211": "$create-and-prepare"
eventstore_1          | [00001,22,07:59:30.691] SLOW BUS MSG ["bus"]: "CreateAndPrepare" - 78ms. Handler: "ProjectionCoreService".
eventstore_1          | [00001,23,07:59:30.691] SLOW BUS MSG ["bus"]: "CreateAndPrepare" - 77ms. Handler: "ProjectionCoreService".
eventstore_1          | [00001,22,07:59:30.691] SLOW QUEUE MSG ["Projection Core #0"]: "CreateAndPrepare" - 78ms. Q: 6/11.
eventstore_1          | [00001,24,07:59:30.691] SLOW BUS MSG ["bus"]: "CreateAndPrepare" - 77ms. Handler: "ProjectionCoreService".
eventstore_1          | [00001,23,07:59:30.691] SLOW QUEUE MSG ["Projection Core #1"]: "CreateAndPrepare" - 77ms. Q: 5/12.
eventstore_1          | [00001,24,07:59:30.691] SLOW QUEUE MSG ["Projection Core #2"]: "CreateAndPrepare" - 77ms. Q: 5/12.
eventstore_1          | [00001,24,07:59:30.726] PROJECTIONS: Scheduling the writing of "$prepared" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,22,07:59:30.726] PROJECTIONS: Scheduling the writing of "$prepared" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,22,07:59:30.726] PROJECTIONS: Command received: 1@"$create-and-prepare"
eventstore_1          | [00001,22,07:59:30.726] PROJECTIONS: Scheduling the writing of "$prepared" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,23,07:59:30.726] PROJECTIONS: Scheduling the writing of "$prepared" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,23,07:59:30.733] PROJECTIONS: Command received: 1@"$create-and-prepare"
eventstore_1          | [00001,24,07:59:30.764] PROJECTIONS: Finished writing events to "$projections-$master": "$prepared"
eventstore_1          | [00001,21,07:59:30.764] PROJECTIONS: Response received: 7@"$prepared"
eventstore_1          | [00001,22,07:59:30.764] PROJECTIONS: Finished writing events to "$projections-$master": "$prepared"
eventstore_1          | [00001,22,07:59:30.764] PROJECTIONS: Finished writing events to "$projections-$master": "$prepared"
eventstore_1          | [00001,21,07:59:30.800] PROJECTIONS: Response received: 8@"$prepared"
eventstore_1          | [00001,21,07:59:30.800] PROJECTIONS: Response received: 10@"$prepared"
eventstore_1          | [00001,21,07:59:30.800] PROJECTIONS: Response received: 11@"$prepared"
eventstore_1          | [00001,23,07:59:30.802] PROJECTIONS: Scheduling the writing of "$prepared" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,23,07:59:30.802] PROJECTIONS: Finished writing events to "$projections-$master": "$prepared"
eventstore_1          | [00001,23,07:59:30.812] PROJECTIONS: Finished writing events to "$projections-$master": "$prepared"
eventstore_1          | [00001,21,07:59:30.812] '"$by_category"' projection source has been written
eventstore_1          | [00001,21,07:59:30.812] PROJECTIONS: Scheduling the writing of "$load-stopped" to "$projections-$38062e7e-582a-4c3b-83dd-71055dd8d4af". Current status of Writer: Busy: false
eventstore_1          | [00001,21,07:59:30.812] '"$streams"' projection source has been written
eventstore_1          | [00001,21,07:59:30.812] '"$by_event_type"' projection source has been written
eventstore_1          | [00001,21,07:59:30.812] '"$stream_by_category"' projection source has been written
eventstore_1          | [00001,21,07:59:30.812] PROJECTIONS: Scheduling the writing of "$load-stopped" to "$projections-$cb6f4bdd-33d3-4865-97d8-3def8bd6ae6d". Current status of Writer: Busy: false
eventstore_1          | [00001,21,07:59:30.812] PROJECTIONS: Scheduling the writing of "$load-stopped" to "$projections-$cb6f4bdd-33d3-4865-97d8-3def8bd6ae6d". Current status of Writer: Busy: true
eventstore_1          | [00001,21,07:59:30.812] PROJECTIONS: Scheduling the writing of "$load-stopped" to "$projections-$5a388580-05b4-472c-ae89-5ac110875211". Current status of Writer: Busy: false
eventstore_1          | [00001,21,07:59:30.812] PROJECTIONS: Response received: 13@"$prepared"
eventstore_1          | [00001,21,07:59:30.827] PROJECTIONS: Finished writing events to "$projections-$38062e7e582a4c3b83dd71055dd8d4af": "$load-stopped"
eventstore_1          | [00001,24,07:59:30.827] PROJECTIONS: Command received: 1@"$load-stopped"
eventstore_1          | [00001,21,07:59:30.835] PROJECTIONS: Finished writing events to "$projections-$cb6f4bdd33d3486597d83def8bd6ae6d": "$load-stopped"
eventstore_1          | [00001,22,07:59:30.835] PROJECTIONS: Command received: 2@"$load-stopped"
eventstore_1          | [00001,21,07:59:30.835] PROJECTIONS: Finished writing events to "$projections-$5a38858005b4472cae895ac110875211": "$load-stopped"
eventstore_1          | [00001,21,07:59:30.835] '"$by_correlation_id"' projection source has been written
eventstore_1          | [00001,21,07:59:30.835] PROJECTIONS: Scheduling the writing of "$load-stopped" to "$projections-$5a388580-05b4-472c-ae89-5ac110875211". Current status of Writer: Busy: false
eventstore_1          | [00001,23,07:59:30.835] PROJECTIONS: Command received: 2@"$load-stopped"
eventstore_1          | [00001,21,07:59:30.844] PROJECTIONS: Finished writing events to "$projections-$cb6f4bdd33d3486597d83def8bd6ae6d": "$load-stopped"
eventstore_1          | [00001,21,07:59:30.844] PROJECTIONS: Finished writing events to "$projections-$5a38858005b4472cae895ac110875211": "$load-stopped"
eventstore_1          | [00001,24,07:59:30.872] PROJECTIONS: Scheduling the writing of "$stopped" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,23,07:59:30.872] PROJECTIONS: Scheduling the writing of "$stopped" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,22,07:59:30.872] PROJECTIONS: Scheduling the writing of "$stopped" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,22,07:59:30.872] PROJECTIONS: Command received: 3@"$load-stopped"
eventstore_1          | [00001,23,07:59:30.872] PROJECTIONS: Command received: 3@"$load-stopped"
eventstore_1          | [00001,22,07:59:30.872] PROJECTIONS: Scheduling the writing of "$stopped" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,23,07:59:30.872] PROJECTIONS: Scheduling the writing of "$stopped" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,22,07:59:30.884] PROJECTIONS: Finished writing events to "$projections-$master": "$stopped"
eventstore_1          | [00001,23,07:59:30.884] PROJECTIONS: Finished writing events to "$projections-$master": "$stopped"
eventstore_1          | [00001,24,07:59:30.884] PROJECTIONS: Finished writing events to "$projections-$master": "$stopped"
eventstore_1          | [00001,21,07:59:30.884] PROJECTIONS: Response received: 18@"$stopped"
eventstore_1          | [00001,21,07:59:30.884] PROJECTIONS: Response received: 21@"$stopped"
eventstore_1          | [00001,21,07:59:30.884] PROJECTIONS: Response received: 24@"$stopped"
eventstore_1          | [00001,22,07:59:30.894] PROJECTIONS: Finished writing events to "$projections-$master": "$stopped"
eventstore_1          | [00001,23,07:59:30.894] PROJECTIONS: Finished writing events to "$projections-$master": "$stopped"
eventstore_1          | [00001,21,07:59:30.894] PROJECTIONS: Response received: 28@"$stopped"
eventstore_1          | [00001,21,07:59:30.902] PROJECTIONS: Response received: 32@"$stopped"
dynamodb_projector_1  | Using local DynamoDB client
adaptor_1             | Using local DynamoDB client
adaptor_1             | Using local DynamoDB client
api_1                 | [2019-04-03 07:59:50] INFO  WEBrick 1.4.2
api_1                 | [2019-04-03 07:59:50] INFO  ruby 2.5.3 (2018-10-18) [x86_64-linux-musl]
api_1                 | [2019-04-03 07:59:50] INFO  WEBrick::HTTPServer#start: pid=1 port=9292
eventstore_1          | [00001,26,07:59:50.948] Segments count: 1, buffers count: 20, should be when full: 20
eventstore_1          | [00001,21,07:59:51.022] Adding projection 48c13d49-c9fa-4b98-b6cc-60137a96192b@"ignore-system-events" to list
eventstore_1          | [00001,21,07:59:51.022] PROJECTIONS: Scheduling the writing of "$create-and-prepare" to "$projections-$38062e7e-582a-4c3b-83dd-71055dd8d4af". Current status of Writer: Busy: false
eventstore_1          | [00001,24,07:59:51.035] PROJECTIONS: Command received: 2@"$create-and-prepare"
eventstore_1          | [00001,21,07:59:51.035] PROJECTIONS: Finished writing events to "$projections-$38062e7e582a4c3b83dd71055dd8d4af": "$create-and-prepare"
dynamodb_projector_1  | Traceback (most recent call last):
dynamodb_projector_1  | 	5: from script/project_to_dynamodb:16:in `<main>'
dynamodb_projector_1  | 	4: from script/project_to_dynamodb:16:in `new'
dynamodb_projector_1  | 	3: from /app/app/projectors/follow_all_events.rb:21:in `initialize'
dynamodb_projector_1  | 	2: from /app/app/projectors/follow_all_events.rb:33:in `create_all_events_stream'
dynamodb_projector_1  | 	1: from /app/lib/event_source_toolkit/http/projection.rb:39:in `produce_non_system_events'
dynamodb_projector_1  | /app/lib/event_source_toolkit/http/projection.rb:22:in `upsert_continuous': #<HTTP::Response/1.1 500 Internal Server Error {"Access-Control-Allow-Methods"=>"GET, POST, OPTIONS", "Access-Control-Allow-Headers"=>"Content-Type, X-Requested-With, X-Forwarded-Host, X-Forwarded-Prefix, X-PINGOTHER, Authorization, ES-LongPoll, ES-ExpectedVersion, ES-EventId, ES-EventType, ES-RequiresMaster, ES-HardDelete, ES-ResolveLinkTos", "Access-Control-Allow-Origin"=>"*", "Access-Control-Expose-Headers"=>"Location, ES-Position, ES-CurrentVersion", "Content-Type"=>"text/plain; charset=utf-8", "Server"=>"Mono-HTTPAPI/1.0", "Date"=>"Wed, 03 Apr 2019 07:59:51 GMT", "Content-Length"=>"75", "Connection"=>"close"}>: An item with the same key has already been added. Key: ignore-system-events (RuntimeError)
eventstore_1          | [00001,24,07:59:51.124] SLOW BUS MSG ["bus"]: "CreateAndPrepare" - 87ms. Handler: "ProjectionCoreService".
eventstore_1          | [00001,24,07:59:51.124] SLOW QUEUE MSG ["Projection Core #2"]: "CreateAndPrepare" - 88ms. Q: 4/7.
eventstore_1          | [00001,24,07:59:51.124] PROJECTIONS: Scheduling the writing of "$prepared" to "$projections-$master". Current status of Writer: Busy: true
eventstore_1          | [00001,24,07:59:51.133] PROJECTIONS: Finished writing events to "$projections-$master": "$prepared"
eventstore_1          | [00001,21,07:59:51.133] PROJECTIONS: Response received: 35@"$prepared"
eventstore_1          | [00001,21,07:59:51.133] '"ignore-system-events"' projection source has been written
eventstore_1          | [00001,21,07:59:51.149] PROJECTIONS: Scheduling the writing of "$start" to "$projections-$38062e7e-582a-4c3b-83dd-71055dd8d4af". Current status of Writer: Busy: false
eventstore_1          | [00001,24,07:59:51.160] PROJECTIONS: Command received: 3@"$start"
eventstore_1          | [00001,21,07:59:51.160] PROJECTIONS: Finished writing events to "$projections-$38062e7e582a4c3b83dd71055dd8d4af": "$start"
eventstore_1          | [00001,24,07:59:51.216] PROJECTIONS: Scheduling the writing of "$started" to "$projections-$master". Current status of Writer: Busy: false
eventstore_1          | [00001,21,07:59:51.225] PROJECTIONS: Response received: 38@"$started"
eventstore_1          | [00001,24,07:59:51.268] PROJECTIONS: Finished writing events to "$projections-$master": "$started"
dynamodb_projector_1  | src_dynamodb_projector_1 exited with code 1
```

**Comment by jageall** [comment]
Possibly fixed by #1963

**Comment by ylorph** [comment]
fixed by #1963
